packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js

```diff
@@ -274,7 +278,16 @@ const CollectionTypeFormWrapper = ({ allLayoutData, children, slug, id, origin }
         dispatch(setStatus('resolved'));
       }
     },
-    [cleanReceivedData, displayErrors, replace, slug, dispatch, rawQuery, toggleNotification]
+    [
+      cleanReceivedData,
+      displayErrors,
+      replace,
+      slug,
+      dispatch,
+      rawQuery,
+      toggleNotification,
+      setCurrentStep,
```

Comment:
Also shouldn't we have the same logic for the single types?

Reply:
So far we had no issue with having it as a dependency 🤔
Good question, I think we assumed it was only collection types but now I have doubts. Since we talk about Single types in the modal I guess you're right! I'll check with the team! @iparidhi @maevalienard